x-pack/auditbeat/module/system/_meta/config.yml.tmpl

```diff
@@ -1,21 +1,33 @@
+{{ if ne .GOOS "windows" -}}
 {{ if .Reference -}}
+# The system module collects security related information about a host.
+# All metricsets send both periodic state information (e.g. all currently
+# running processes) and real-time changes (e.g. when a new process starts
+# or stops).
 {{ end -}}
-{{ if ne .GOOS "windows" -}}
 - module: system
-
   metricsets:
-    - host
+    - host    # General host information, e.g. uptime, IPs
     {{ if false -}}
-    - packages
+    - packages # Installed packages
     {{- end -}}
-    - process
+    - process # Started and stopped processes
     {{ if eq .GOOS "linux" -}}
-    - socket
-    - user
+    - socket  # Opened and closed processes
+    - user    # User information
```

Comment:
"User information and groups"? The groups are not listed separately, they're only reported as part of the user, so I think this would be more accurate.

Reply:
Cool that's fine